src/lib/snack-bar/base-snack-bar.ts

```diff
@@ -0,0 +1,12 @@
+import {MdSnackBarRef} from './snack-bar-ref';
+
+
+export class BaseSnackBarContent<T> {
```

Comment:
What's this base class for, since it's only ever inherited by from `SimpleSnackbar`?